test/GCClassic/integration/integrationDiffTest.sh

```diff
@@ -0,0 +1 @@
+../../shared/integrationDiffTest.sh
```

Comment:
I wonder if we should have test/difftest, test/integration, and test/parallel folders and then have subfolders for GCHP and GC-Classic in those. This would avoid the symlinks to shared which is kind of confusing. Also, the difftest can be applied to parallelization or integration tests so it might make more sense to have it in its own folder.